### PR TITLE
Feature/login style updates

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

I updated styling changes to the Login and SignUp. After meeting with Ryan Hamblin, we realized the Okta widget needed some work on the backend admin side before we could change the input from 'Username' to 'Email Address'. We also cannot change the drop-down help section just yet. I will follow-up with him in a week to check the status. I also hide the logo on the Login widget and instead implemented it in the return. This helped style it to match the Sign-up page.

We also decided that when you go back to the login page from the Sign-up page, it is ok that it rerenders. This fixed the error and at least works so it is not a high priority to fix. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -50,20 +49,23 @@ const LoginContainer = () => {
...
-          <Link to="/signup">
-            <p>Request to join</p>
+      <div className="form">
+        <div className="logo">
+          <img src={logo} alt="logo" width="350px" />
+        </div>
+        <div className="login-page">
+          <div id="sign-in-widget" aria-label="login form" />
+          <Link className="link-styles" to="/signup">
+            <p>Don't have an account? Register here</p>
```
Comment (MichelleCCodes) [inline src/components/pages/Login/LoginContainer.js:64]: This message is such a good improvement from what we had before!

**Review by MichelleCCodes — APPROVED** [review]
Styling changes look good! I noticed there are two styling pages for Login in, one in CSS and one in Less. Will both files need to be used?

File: src/components/pages/Login/LoginContainer.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React, { useEffect } from 'react';
 import { Link } from 'react-router-dom';
-import { StyledLogin } from './LoginContainerStyled';
+// import { StyledLogin } from './LoginContainerStyled';
+import './_LoginContainerStyled.less';
```
Comment (paulstgermain) [inline src/components/pages/Login/LoginContainer.js:4]: Nice - loving the changeover to LESS and steady removal of MUI!

File: src/components/pages/Login/LoginContainer.js (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/Login/LoginContainer.js:64]: Seconding this - and this code is much cleaner/more involved than what we had before!

File: src/components/pages/Login/_SignupPageStyled.css [hunk 3]
```diff
@@ -2,10 +2,22 @@
...
 }
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 50%;
+  margin: 0 auto;
+}
+@media (max-width: 600px) {
```
Comment (paulstgermain) [inline src/components/pages/Login/_SignupPageStyled.css:13]: Love the forethought into responsiveness by adding another media query!

**Review by paulstgermain — APPROVED** [review]
Nice work moving everything over into LESS and further fine-tuning the styles on the login and signup pages. I see no problems with this code and everything clean, so... approved!

**Pull request merged**